### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by honst**
解决找不到mysqldump

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ MAINTAINER siddontang
 
 COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
 
-RUN cd /go/src/github.com/siddontang/go-mysql-elasticsearch/ && \
+RUN apk add --no-cache mariadb-client && \
```
Comment (siddontang) [inline Dockerfile]: can you use another RUN for apk especially?

**Comment by siddontang** [comment]
Rest LGTM

File: Dockerfile (hunk 1 above)
Comment (honst) [reply Dockerfile]: Yes.
I thought the mysqldump is required by go-mysql-elasticsearch, so made them into one layer.

Comment (siddontang) [reply Dockerfile]: Mostly, installing pkg should be separated from others. It is cache friendly for docker.

**Comment by siddontang** [comment]
already included, close now. 

Thanks @honst

**Pull request closed without merging**